stress/shell/src/main/java/alluxio/stress/cli/GetPinnedFileIdsBench.java

```diff
@@ -67,13 +67,15 @@ public String getBenchDescription() {
         "A benchmarking tool for the GetPinnedFileIds RPC.",
         "The test will generate a specified number of test files and pin them. "
             + "Then it will keep calling the GetPinnedFileIds RPC by the specified load until the "
-            + "specified duration has elapsed.",
+            + "specified duration has elapsed. The test files will be cleaned up in the end.",
         "",
         "Example:",
-        "4 simulated workers running on 2 job workers, requesting a total number of 100k pinned "
-            + "files for a total of 5 seconds:",
-        "$ bin/alluxio runClass alluxio.stress.cli.GetPinnedFileIdsBench --concurrency 2 "
-            + "--cluster-limit 2 --num-files 100000 --duration 5s",
+        "# 2 job workers will be chosen to run the benchmark",
+        "# Each job worker runs 3 simulated clients",
+        "# Each client keeps requesting a total number of 10k pinned files for "
+            + "a total of 100 milliseconds",
+        "$ bin/alluxio runClass alluxio.stress.cli.GetPinnedFileIdsBench --concurrency 3 \\",
+        "--cluster --cluster-limit 2 --num-files 10000 --duration 100ms",
```

Comment:
Since this does not actually work now (needs `--in-process` instead of `--cluster`), IMO we should provide a working example and change it after the blocker is fixed.